packages/flutter_tools/lib/src/commands/build_flx.dart

```diff
@@ -12,6 +12,7 @@ import 'build.dart';
 class BuildFlxCommand extends BuildSubCommand {
   BuildFlxCommand({bool verboseHelp: false}) {
     usesTargetOption();
+    usesPackagesOption();
```

Comment:
I believe we can also accomplish that by creating a new command that doesn't include the pubspec.yaml validation step.